CHANGES.rst

```diff
@@ -1,6 +1,11 @@
 Changes
 =======
 
+Version 0.9.4 (UNRELEASED)
+--------------------------
+
+- Fixes the deployment of CVMFS by setting the correct namespace on the related PersistentVolumeClaim.
```

Comment:
What about the following change in order to express that this fix was not general for everybody, but only applying to specific deployments?

- Fixes the mounting of CVMFS volumes for the REANA deployments that use non-default Kubernetes namespace.